recipes/torchtime/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "torchtime" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchtime-{{ version }}.tar.gz
+  sha256: 45ce7d95fa9974f0c6edb967cf08546f68b00e49d0f997a6a69281900f3de886
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.8,<3.10
+    - pip
+    - poetry >=1
+  run:
+    - python >=3.8,<3.10
+    - sktime >=0.12.1,<1.13
+    - requests >=2.27.1,<2.28
+    - scikit-learn >=1.1.1,<1.2
+    - tqdm >=4.64.0,<4.65
+    - pytorch  # pytorch is globally pinned
+    - numpy >=1.21.0,<1.22
```

```suggestion
    - python >=3.8,<3.10
    - sktime >=0.12.1,<2
    - requests >=2.27.1,<3
    - scikit-learn >=1.1.1,<2
    - tqdm >=4.64.0,<5
    - pytorch >=1.11.0,<2
    - numpy >=1.21.0,<2
```
I was mistaken. Pytorch is actually only pinned for packages that link against it, but yours isn't compiled, so you can list the version compatibility. 

Also, this package poetry dependency spec is using caret not tilde.

https://github.com/philipdarke/torchtime/blob/7feb206557905282aba4b7bcbf5ab939794b7972/pyproject.toml#L13

https://python-poetry.org/docs/dependency-specification/